### DISCUSSION TITLE
New York time square display ad

### DISCUSSION THREAD

**Discussion by syavash** [WeAreMahsaAmini/Core#11] [Ideas]
We can purchase an ad spot for a day in time square for $5-$50k. Shall we raise money and go for it?

**Comment by triplea24** [comment]
@syavash Absolutely. Nothing gets more exposure than ad in a billboard in important places:
1. Times Square, New York City
2. Young-Dundas Square, Toronto
3. London
4. Paris
5. Berlin

We can also go for smaller billboard advertising. Like bus advertising, etc.

**Comment by Pouria-007** [comment]
**Sounds great! We can create a Crowdfunding account:** 

![image](https://user-images.githubusercontent.com/114555071/192697081-17ac43b2-ae63-48a2-94af-991719923e46.png)
![image](https://user-images.githubusercontent.com/114555071/192697223-7fbc9be3-6d0e-4a05-8d9e-ee5924d74528.png)

**Comment by bidar-ca** [comment]
Any update?

Please review my idea at #209

**Comment by SinaKarimi7** [comment]
@syavash Is there any update or progress on this idea?
